src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/ConfigReader.java

```diff
@@ -0,0 +1,36 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.deciders.Decider;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class ConfigReader {
+    private static final Logger LOG = LogManager.getLogger(ConfigReader.class);
+    private static HashMap<Decider, ArrayList<String>> deciderActionPriorityOrder;
+
+    public static final String CONFIG_FILE_PATH = "./pa_config/deciderConfig.yml";
```

Comment:
Is this is the right way to provide file path for configuration files? What is the current directory here for `./...`?
Please see how path has been specified for rca.conf in `RcaConsts.java` and change accordingly.